.github/workflows/release-periodically.yml

```diff
@@ -61,8 +61,14 @@ jobs:
         if: steps.check-tag.outputs.can_bump_version
         # the repo should have an access token from the checkout action
         # https://github.com/actions/checkout#Push-a-commit-using-the-built-in-token
+        # push first the commit to ensure there isn't anything merged before starting, and then add just the new tag
```

Comment:
I think this comment is not correct anymore.

Reply:
Thanks! good catch, fixed.